README.md

```diff
@@ -1 +1,431 @@
-# Reference Guide: Objective-C Fundamentals
\ No newline at end of file
+# Reference Guide: Objective-C Fundamentals
+
+## Table of Contents
+
+* [NSLog Basics](#nslog-basics)
```

Comment:
that's actually a primary concern i have. should we add a note telling them to utilize this document via Github? that feels icky too.